docs/tutorial.md

```diff
@@ -0,0 +1,275 @@
+| `--datatype`       | Datatype flag helps CytoSnake in how to setup the input files for processing. \[Choices = "cell_profiler" "deep_profiler"\] \[Default="cell_profiler"\] |
+
+### `run` mode documentation
+
+Here are the list of parameters that CytoSnake's `run` mode currently
+supports
+
+| Parameters         | Documentation                                                                                                                                                                                |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Required Arguments |                                                                                                                                                                                              |
+| workflow           | Name of the workflow to execute                                                                                                                                                              |
+| Optional Arguments |                                                                                                                                                                                              |
+| `-c` `--max_core`  | Maximum number of cores to use for the workflow default=1                                                                                                                                    |
+| `--lock`           | Directory becomes locked when workflow is executed. if any interruptions has occurred, if True, the directory will be automatically unlocked, else, it will remain locked. Default is False. |
+| `--force`          | Force re-run of the workflow. This means generated files will be over-written with the outputs produced from the forced re-run                                                               |
+
+## Usage
+
+### Download data
+
+In this usage tutorial, we will be using data cell health datasets.
+
+You can download these datasets (quite large files):
+
+- [plate_data_1]: (10GB download)
+- [plate_data_2]: (11GB download)
+- [metadata_folder]: Contains all associated perturbations per well
+- [barcode]: Maps plate id with plate names
+
+You can also use your dataset but some of the tasks that are being done
+here are specific to the files downloaded.
+
+### Setting up files
+
+If you are using the downloaded dataset. unzip the zip files in the
+directory where the CytoSnake source is.
+
+```text
+unzip metadata.zip && unzip barcode_platemap.csv.zip
+```
+
+The first step it to prepare your files for analysis, this is simply
+executed by typing:
+
+```text
+cytosnake init -d SQ00014613.sqlite SQ00014613.sqlite -d metadata -b barcode_platemap.csv
+```
+
+In instances where you may have a lot of data, CytoSnake supports
+wildcard variables.
```

Comment:
This will hold true for all parameters as `CytoSnake` uses `argparse` as it's argument parser. `argparse` can take in wildcard parameters. Since `cytosnake`'s `init` mode currently expects file paths, then all parameters should have wildcard support. 

Hopefully this makes sense!

Reply:
@axiomcura Understood - thanks for clarifying! Based on this, would it be beneficial at all to note that the wildcard argument parsing is mostly intended for the datafiles (and not the metadata)?

Reply:
@d33bs that's a good point!